### PR TITLE
Bono/goth 235b

### PR DESCRIPTION
**Pull request by bonomite**
* unfortunately dude to my formatting, we see a lot of noise in this PR

- globally using "getArticleImage" helper function and updating the function to inject quality
- formatting most images to a 3:2 ratio with no scaling in browser
- responsive image size loading
- gallery image size based on browser width
- adjustments for a general 640px width in content for home page
- setting width/heights to match it's desktop render size
- DS update

Also, takes care of ticket: https://nypublicradio-digital.atlassian.net/browse/GOTH-250

removed the HTL hack SCSS in the default layout

### CODE REVIEW

**Review by kimlarocca — APPROVED** [review]
awesome!

File: pages/_section/photos/_gallery.vue [hunk 1]
```diff
@@ -141,9 +143,27 @@ export default {
...
+
+    const IMG_SCALE_POINT = 1147
+    const IMG_SCALE_MAX = 1098
+
+    const PADDING = isMobileDevice ? 32 : 49
+    const WINDOW_WIDTH = window.innerWidth
+
+    this.imgWidth =
+      WINDOW_WIDTH < IMG_SCALE_POINT ? WINDOW_WIDTH - PADDING : IMG_SCALE_MAX
+  },
```
Comment (walsh9) [inline pages/_section/photos/_gallery.vue:159]: i don't understand why we're trying to do device detection here? Is there a reason to not just use the viewport width?
Also if we're using imgWidth in the template we should add it to the component data object and maybe give it a more descriptive name.

**Review by walsh9 — APPROVED** [review]
Looks good, but on thing stood out to me so i put a question.

File: pages/_section/photos/_gallery.vue (hunk 1 above)
Comment (bonomite) [reply pages/_section/photos/_gallery.vue:154]: yeah... nice... simplified this.

**Pull request merged**